dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -476,5 +476,10 @@ error.root.must.be.server=CWWKG0105E: The {0} configuration document must begin
 error.root.must.be.server.explanation=The configuration must be contained in either a server element or a client element. 
 error.root.must.be.server.useraction=Add a server or client element as the root of the configuration file. 
 
+error.bad.variable.file=CWWKG0106E: The file {0} can not be read. 
```

Comment:
**change**
`The file {0} can not be read. `
**to**
`The {0} file can not be read.`